### DISCUSSION TITLE
有没有打算做像Lottielab这样的编辑器，AE真的很难用

### DISCUSSION THREAD

**Discussion by 1805839492** [Tencent/libpag#2594] [Ideas] [answered]
Lottielab链接[https://www.lottielab.com/dashboard/team/99b04aa5-c26b-4c79-b673-13445a7a4246](url)

**Comment by domchen** [comment, marked as answer]
官方暂时没有这方面打算。LottieLab也是第三方的，不是Lottie自己做的。第三方要是感兴趣，也可以围绕PAG开源的方案去探索自定义编辑器。
